aztk_cli/utils.py

```diff
@@ -463,12 +463,12 @@ def log_property(label: str, value: str):
     log.info("{0:30} {1}".format(label, value))
 
 
-def log_execute_result(node_id, result):
-    log.info("-" * (len(node_id) + 4))
-    log.info("| %s |", node_id)
-    log.info("-" * (len(node_id) + 4))
-    if isinstance(result, Exception):
-        log.info("%s\n", result)
+def log_node_output(node_output):
+    print("NodeOutput", node_output.output)
```

Comment:
print?

Reply:
ah this is a debug statement, will remove. thanks!